src/interpreter/naive_interpreter.nit

```diff
@@ -314,6 +314,15 @@ class NaiveInterpreter
 		return frames.first.arguments.first.mtype.as(MClassType)
 	end
 
+	# Initialize the environment for a call and return a new Frame
+	# *`node` The AST node
+	# *`mpropdef` The corresponding mpropdef
+	# *`args` Arguments of the call
+	fun initialize_frame(node: APropdef, mpropdef: MPropDef, args: Array[Instance]): Frame
```

Comment:
why `initialize`? I fact there is no that much initialization done. It is more a factory method. What about `new_frame` ? the `new_` prefix seems to be a common way in Nit to name factory methods.